woocommerce/woocommerce-admin
# Add tracks to import jobs

Fixes #2173.

Add tracks to import and delete imported data jobs.

### Detailed test instructions:
- Go to _Settings_ and press _Rebuild reports_.
- Make a call to `/wc/v4/reports/import/delete` (or merge #2198 and press _Delete Previously Imported Data_).
- Wait some minutes and go to Tracks Live View. There, verify the events appear, and they have the `type` in the properties.
  - https://mc.a8c.com/tracks/live/?eventname=wcadmin_import%25&user=&useragent=
  - https://mc.a8c.com/tracks/live/?eventname=wcadmin_delete%25&user=&useragent=
